Add list of Insight attributes (#3344)
SumoLogic/sumologic-documentation: docs/cloud-soar/incidents-triage.md

still become `opt_9`. It is important to remember these field numbers, as
they will be used when the API is invoked.

#### Attributes sent from Cloud SIEM

You can ingest Cloud SIEM Insights into Cloud SOAR for incident triage using the `GetInsight` Cloud SOAR API. The following Insight attributes are returned. 

When you create an incident from an Insight, you can map the Insight attributes to fields in Cloud SOAR as follows:

| Attribute in Cloud SIEM | Field in Cloud SOAR | 
| :-- | :-- | 
| `assignee` | `Insight Assignee` (custom field) |
| `created` | `Start time` |
| `description` | `Additional Info` |
| `entity.value` | `Primary Entity` (custom field) |
| `entity.type` | `Entity Type` (custom field) |
| `id` | `Insight ID` (custom field) |
| `involvedEntities[].value` | `Involved Entities` (custom field) |
| `readableId` | `Incident ID` (custom field) |
| `severity` | `Severity` |
| `status.displayName` | `Status` |
| `tags[]` | `Tags` |

:::note

> You cannot add additional required attributes to the incident template when creating incidents from Insights, or an error will result. Only those attributes sent over with Insights can be used as required attributes on the template. 

Reviewer: ```suggestion
When creating incidents from Insights, adding additional required attributes to the incident template will result in an error. Only those attributes sent over with Insights can be used as required attributes on the template. 
```
